fetchai/ledger/parser/etch.grammar

```diff
@@ -18,41 +18,41 @@ instruction: "use" NAME [use_shard] ";"                         -> use_global
 
 instantiation: "var" NAME "=" expression ";"
```

Comment:
Not part of these changes, but `"var" typed_name "=" expression ";"` is also valid in Etch.

Reply:
Fixed + added test.